toolchain/check/thunk.cpp

```diff
@@ -327,37 +317,58 @@ static auto BuildPatternRef(Context& context, SemIR::FunctionId function_id,
   return pattern_ref_id;
 }
 
-// Build a call to a function that forwards the arguments of the enclosing
-// function, for use when constructing a thunk.
```

Comment:
This comment appears to be deleted; the new copy doesn't have a comment. Would it be worth retaining?

Reply:
Done.